lib/rspec/core/reporter.rb

```diff
@@ -79,6 +88,17 @@ def message(message)
       notify :message, Notifications::MessageNotification.new(message)
     end
 
+    # @param event [Symbol] Name of the custom event to trigger on formatters
+    # @param options [Hash] Hash of arguments to provide via `CustomNotification`
+    #
+    # Publish a custom event to supporting registered formatters.
+    # @see RSpec::Core::Notifications::CustomNotification
+    def publish(event, options={})
```

Comment:
Personally, no, I like the conciseness of the name we've never supported directly sending those events and most people won't even be aware you can so I think as a new API this is fine :)